File: commentApp/src/com/itbank/comments/model/Comments.java
package com.itbank.comments.model;

public class Comments {
	private int comments_id;
	private int board_id;
	private String cwriter;
	private String msg;
	private String cregdate;

	public int getComments_id() {
		return comments_id;
	}

	public void setComments_id(int comments_id) {
		this.comments_id = comments_id;
	}

	public int getBoard_id() {
		return board_id;
	}

	public void setBoard_id(int board_id) {
		this.board_id = board_id;
	}

	public String getCwriter() {
		return cwriter;
	}

	public void setCwriter(String cwriter) {
		this.cwriter = cwriter;
	}

	public String getMsg() {
		return msg;
	}

	public void setMsg(String msg) {
		this.msg = msg;
	}

	public String getCregdate() {
		return cregdate;
	}

	public void setCregdate(String cregdate) {
		this.cregdate = cregdate;
	}
	
}

File: commentApp/WebContent/board/Comments.js
class Comments{
	constructor(comments_id, cwriter, msg)	{
		this.str="";
		this.str+="<div class=\"reply\">";
		this.str+="<div id=\"cwriter\">"+cwriter+"</div>";
		this.str+="<div id=\"msg\">"+msg+"</div>";
		this.str+="</div>";
	}
	getStr(){
		return this.str;
	}
}